src/event_object_movement.c

```diff
@@ -2077,6 +2079,7 @@ void UpdateFollowingPokemon(void)
     // 3. flag is set
     if (OW_FOLLOWERS_ENABLED == FALSE
      || !GetFollowerInfo(&species, &form, &shiny)
+     || SpeciesToGraphicsInfo(species, 0) == NULL
```

Comment:
```suggestion
     || SpeciesToGraphicsInfo(species, form) == NULL
```

Reply:
Shouldn't the line below also use `form` over 0, in that case?

Reply:
Yes, please update that as well.